docs/workflows/push.rst

```diff
@@ -3,16 +3,16 @@
 Push content to a Repository
 =============================
 
-Users can push a container image to the repositories hosted by the Container Registry
-
-.. note::
-   If token auth is enabled admin credentials will be required during the push operation.
-   Provide them in the login to the registry or in each API call.
+Users can push images to the repositories hosted by the Container Registry. Only the users who are
+logged in to the registry are allowed to perform such an action. Find below a complete example of
+pushing a tagged image.
```

Comment:
@lubosmj  can you add a note that will say that if you have toke-auth disabled only admin user can push.
i am not sure where we say that in some other place?

Reply:
Oh, I am sorry. Other users will be allowed to push images after resolving the story https://pulp.plan.io/issues/8271, right?

Reply:
right, for now we have enabled only admin to push if you have disabled auth.